modules/syslog-configuring-acs.adoc

```diff
@@ -17,5 +17,6 @@ Create a new syslog integration in {product-title}.
 . If you are using TLS, turn on the *Use TLS* toggle.
 . If your syslog receiver uses a certificate that is not trusted, turn on the *Disable TLS Certificate Validation (Insecure)* toggle.
 Otherwise, leave this toggle off.
+. Click *Add new extra field* to add extra fields. For example, if your syslog receiver accepts objects from multiple sources, type `source` and `rhacs` in the *Key* and *Value* fields, and filter on this value to identify all alerts from {product-title-short}.
 . Select *Test* (`checkmark` icon) to send a test message to verify that the integration with your generic webhook is working.
-. Select *Create* (`save` icon) to create the configuration.
+. Select *Create* (`save` icon) to create the configuration.
```

Comment:
I don't see a change on this line. Not sure why it's included in the automatic diff